weather/schemas.py

```diff
@@ -0,0 +1,21 @@
+from pydantic import BaseModel
+
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: str
```

Comment:
The `additional_info` field is defined as a required string. If this field is optional, consider setting it to `Optional[str] = None` to reflect its optional nature.